ports/slikenet/vcpkg.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "slikenet",
+  "version-string": "2021-06-07",
```

```suggestion
  "version-date": "2021-06-07",
```
This should be `version-date` here. Please see the doc here. https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-date